lte/gateway/deploy/roles/magma_test/tasks/main.yml

```diff
@@ -87,14 +76,20 @@
 
 - name: Install MySQL client for OAI upstreaming
   become: yes
-  apt: pkg=default-libmysqlclient-dev state=present update_cache=yes
+  apt:
+    state: present
+    update_cache: yes
+    pkg:
+      - default-libmysqlclient-dev
   when: preburn
 
-- name: Check whether swagger.yml exists
-  stat:
-    path: "{{ swagger_spec }}"
-  register: swagger_yml_present
-  when: full_provision
+- name: Install rules_pyvenv dependency
```

Comment:
I guess this step can be executed in preburn for the next base image. A reminder would be nice.
```suggestion
# TODO: move to preburn
- name: Install rules_pyvenv dependency
```

Reply:
added

Reply:
Note this is already on master in https://github.com/magma/magma/blob/07beeefa252538c1e43e48455775f176a7db27e1/lte/gateway/deploy/roles/dev_common/tasks/main.yml#L198-L206

Reply:
oh, thank you - I will remove this from the magma_test specific role.

Reply:
removed - just to be sure an integ test run to see if the environment still works correctly https://github.com/nstng/magma/actions/runs/3411611189